src/RequestInformation.cs

```diff
@@ -130,7 +130,7 @@ public void AddQueryParameters(object source)
         /// <summary>
         /// The Request Body.
         /// </summary>
-        public Stream Content { get; set; }
+        public Stream Content { get; set; } = Stream.Null;
```

Comment:
I do think we have code that checks for nullability in the implementation. This is just a callout to double check.